OracleSiebel/OEL/build/multi-stage/cache/cache_install_base.sh

```diff
@@ -0,0 +1,6 @@
+chown -R siebel /mnt/Siebel_Enterprise_Server
```

Comment:
You can combine this into a single command:
```
chown -R siebel:siebel /mnt/Siebel_Enterprise_Server siebel
```

Reply:
Done. Thanks